docs/migrate/migrate-estimator.rst

```diff
@@ -0,0 +1,201 @@
+Calculate expectation values in an algorithm
+==============================================
+
+The Estimator primitive serves a double purpose here:
+
+* Acts as entry point to quantum systems and simulators by replacing `backend.run()`  (in the case of the `qiskit-ibm-runtime.Estimator`), as well as the tools we had for exact computations (local) that we can now replace with the `qiskit.primitives.Estimator`.
+* Acts as an **algorithmic abstraction**. Recall that to be executable in a real device, the expectation value calculation must represented as a single quantum circuit. This can be done by a series of operations that were previously left to the user. Most users relied on a series of Qiskit Terra tools  (in `qiskit.opflow`) to perform these operations, so this method will be shown in the legacy implementation section.  
+
+  The new method is much shorter and most of the tools from `qiskit.opflow` used in these examples have been rendered obsolete by the primitives. 
```

Comment:
```suggestion
The role of the `Estimator` primitive is two-fold: on one hand, it acts as an entry point to the quantum devices or simulators, replacing `backend.run()` (or `QuantumInstance.execute()`). On the other hand, it is an **algorithmic abstraction** for expectation value calculations, which removes the need to perform operations to construct the final expectation circuit. This results in a considerable reduction of the code complexity, and a more compact algorithm design. 

The following example uses common tools from the ``qiskit.opflow`` module as a reference for the "legacy way of doing things", but we acknowledge that this is not the only path to perform expectation value calculations. Regardless of what you starting point is, the migration to the `Estimator` primitive will look the same.

```

Reply:
I am not 100% sold on the wording of the last sentence, hopefully you can find a better-sounding way of transmitting the message.